packages/react-native-editor/__device-tests__/gutenberg-editor-rendering-media-blocks.test.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Internal dependencies
+ */
+import { mediaBlocks } from '../src/initial-html';
+
+describe( 'Gutenberg Editor Rendering Blocks test', () => {
```

Comment:
I notice this is the same description used in the other files (i.e. `gutenberg-editor-rendering-other-blocks.test.js`  and `gutenberg-editor-rendering-text-blocks.test.js). I'm wondering if we could either unify the different tests in a single file using the same `describe` or update the title used in each `describe`.

Reply:
I think I'd prefer to update the title of each describe, because having the same test file would require removing the editor content for each case, since we don't have a stable way of doing that I think it could add more flakiness, e.g. selecting the text, deleting it to be able to set a new HTML content...

Reply:
I've updated them in https://github.com/WordPress/gutenberg/pull/47814/commits/bced25f3d72e7714a495f6eefe1b9da8952c5751 let me know what you think, thanks!